index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Agasha Bornittah Portifolio</title>
+    <link rel="stylesheet" href="assets/fonts/css/all.min.css">
+    <link rel="stylesheet" href="assets/fonts/css/brands.min.css">
+    <link rel="stylesheet" href="assets/fonts/css/fontawesome.min.css">
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+   <div class="container">
+       <div class="nav">
+        <h1>Welcome</h1>
+        <div class="nav-menu" >
+            <ul>
+                <li>
+                    <a href="">Portifolio</a>
+                </li>
+                <li>
+                    <a href="">About</a>
+                </li>
+                <li>
+                    <a href="">Contact</a>
+                </li>
+            </ul>
+        </div>
+        <a href="" class="menu-bars">
+            <i class="fas fa-bars"></i>
+        </a>
+       </div>
+        <div class="section">
+          <div class="wrapper">
+            <div class="titles">
+                <h1>Hey there!, I'm Bornittah</h1>
+                <h1 class="secondary">I'm a software developer</h1>
+            </div>
+            <div class="description">
+                <p>
+                    I can help you build a product , feature or website Look through some of my work and
+                    experience! If you like what you see and have a project you need coded,don’t hestiate to contact me.
+                </p>
+            </div>
+            <div class="social-icons">
+                <a href="https://github.com/Bornittah">
+                    <i class="fa-brands fa-github"></i>
+                </a>
+                <a href="https://www.linkedin.com/in/agasha-bornittah-47b594131/">
+                    <i class="fa-brands fa-linkedin"></i>
+                </a>
+                <a href="#">
+                    <i class="fa-brands fa-angellist"></i>
+                </a>
+                <a href="https://twitter.com/AgashaBornittah">
+                    <i class="fa-brands fa-twitter"></i>
+                </a>
+                <a href="https://medium.com/@bornittah">
+                    <i class="fa-brands fa-medium"></i>
+                </a>
+            </div>
```

Comment:
- [ ] Kindly wrap all the social media links in a ul tag instead of a div and each social media link (anchor tag) in an li tag 👍🏿